dist/mk-docker-opts.sh

```diff
@@ -72,23 +72,30 @@ if [ -n "$FLANNEL_IPMASQ" ] && [ $ipmasq = true ] ; then
 	elif [ "$FLANNEL_IPMASQ" = false ] ; then
 		DOCKER_OPT_IPMASQ="--ip-masq=true"
 	else
-		echo "Invalid value of FLANNEL_IPMASQ: $FLANNEL_IPMASQ" > /dev/stderr
+		echo "Invalid value of FLANNEL_IPMASQ: $FLANNEL_IPMASQ" >&2
 		exit 1
 	fi
 fi
 
 eval docker_opts="\$${combined_opts_key}"
-docker_opts+=" "
+
+if [ "$docker_opts" ]; then
+	docker_opts="$docker_opts ";
+fi
 
 echo -n "" >$docker_env
-for opt in $(compgen -v DOCKER_OPT_); do
-	eval val=\$$opt
 
-	if [ "$indiv_opts" = true ]; then
-		echo "$opt=\"$val\"" >>$docker_env
+for opt in $(set | grep "DOCKER_OPT_"); do
+
+    OPT_NAME=$(echo $opt | awk -F "=" '{print $1;}');
```

Comment:
Nice catch @eyakubovich.
Looks like a typical case of space vs. tab indentation.